### PR TITLE
Prevent noop price send

### PR DESCRIPTION
**Pull request by monstrobishi**
#### What kind of PR is this?:

/kind fix

#### What this PR does / why we need it:

It seems that we were sending no-op tx with empty price list (Essentially no effect), this prevents this and adds some additional useful logging.

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/salmon-oracles-functions/__tests__/basic_oracle.test.ts [hunk 1]
```diff
@@ -88,4 +88,25 @@ describe('basic price oracles', () => {
...
+    const broadcastMock = jest.fn(broadcast)
+
+    const oraclesManager = new OraclesManager(
+      broadcastMock,
+      providers.ellipticPair,
+      providers.fee,
+      providers.prevout
+    )
+
+    await oraclesManager.updatePrices('', [])
```
Comment (jingyi2811) [inline packages/salmon-oracles-functions/__tests__/basic_oracle.test.ts:108]: I feel like there is something wrong in this line of code. Ignore me if you think it is okay.

Comment (monstrobishi) [reply packages/salmon-oracles-functions/__tests__/basic_oracle.test.ts:108]: it's just the test, that's intentional here

**Review by fuxingloh — APPROVED** [review]
I think if you not planning to use `updatePrices` `return` value maybe should just keep to void

**Comment by monstrobishi** [comment]
> I think if you not planning to use `updatePrices` `return` value maybe should just keep to void

I think it's important to have a reference to the txid though otherwise there's no way to track what was sent out in production

**Review by jingyi2811 — APPROVED** [review]

**Pull request merged**